src/commands/dev/edge/server/http.rs

```diff
@@ -47,7 +48,7 @@ pub async fn http(
                 let req_method = parts.method.to_string();
                 let now: DateTime<Local> = Local::now();
                 let path = get_path_as_str(&parts.uri);
-                async move {
+                let resp = async move {
```

Comment:
err actually I don't think this does what I want because `?` returns from the whole closure, not just the async block. Let me fix that.